packages/manager/src/features/CloudPulse/shared/CloudPulseRegionSelect.tsx

```diff
@@ -67,5 +64,9 @@ export const CloudPulseRegionSelect = React.memo(
         value={selectedRegion}
       />
     );
-  }
+  },
+  (
+    oldProps: CloudPulseRegionSelectProps,
+    newProps: CloudPulseRegionSelectProps
+  ) => oldProps.selectedDashboard?.id === newProps.selectedDashboard?.id
```

Comment:
Nit: Types are inferred
```suggestion
    prevProps,
    nextProps
  ) => prevProps.selectedDashboard?.id === nextProps.selectedDashboard?.id
```